src/components/header/renderer.tsx

```diff
@@ -141,6 +143,38 @@ class Header extends React.Component<Props, State> {
     this.setState({ fullscreen: event.target.checked });
   }
 
+  public handleHelpModalOpen(event) {
+    // Handle key press in Mac
+    if (event.keyCode === 222 && event.metaKey && !event.shiftKey) {
+      // 222: Apostrophe(') key
+      this.setState({
+        helpModalOpen: true,
+      });
+    }
+
+    // Handle key press in PC
+    if (event.keyCode === 191 && event.ctrlKey && event.shiftKey) {
+      // 191 Slash(/) key
+      this.setState({
+        helpModalOpen: true,
+      });
+    }
+  }
+
+  public handleHelpModalCloseClick() {
+    this.setState({
+      helpModalOpen: false,
+    });
+  }
+
+  public handleHelpModalCloseEsc(event) {
```

Comment:
This functionality is built into the modal. Please use that.

Reply:
@domoritz I couldn't figure out how can I add custom keyboard events to toggle Portal that's why I wrote a custom one. I'll see if I can use `<Portal />` and implement this.

Reply:
Portal has a property to close on escape. We already use portals in the editor.

Reply:
@domoritz Unlike the other modals, this modal is being triggered by a shortcut key. Rest of them were triggered using click only. `onClick` prop would simply trigger the portal. I can't pass a prop which will get triggered when I press `Ctrl + ?`.

Reply:
There must be a way. https://codepen.io/nagyadam2092/pen/eGRxQv is an example that uses setState to open the modal.

Reply:
I have done the same thing. Even I used `setState` to open the modal. Plus in this example, it doesn't close when pressed `Esc`. So I think this will require writing another function.

Reply:
Maybe. I don't have the bandwidth to look deeper into this right now.

Reply:
Understandable :smile: . I'll try using `<Portal />` once and see if it's possible.